commands/http/client.go

```diff
@@ -183,78 +184,41 @@ func getResponse(httpRes *http.Response, req cmds.Request) (cmds.Response, error
 
 	res.SetCloser(httpRes.Body)
 
-	if len(httpRes.Header.Get(streamHeader)) > 0 {
-		// if output is a stream, we can just use the body reader
-		res.SetOutput(httpRes.Body)
+	if contentType != "application/json" {
```

Comment:
there are constants for these... a simple typo could cause many headaches. let's have the type system help us.